### ISSUE TITLE
Move normative definition of Verification Methods and Controller Documents to Data Integrity

### ISSUE THREAD

**Issue by msporny** [w3c/did-core#845]
DID Core currently defines Verification Methods and Controller Documents, which was a stop-gap measure until we were able to start the Data Integrity work at W3C. Now that that work exists, and the normative definitions are over there, we should point the DID Core definitions for Verification Methods and Controller Documents to Data Integrity (in the next version of the specification). To be clear, this would be an Editorial change requiring no normative updates to implementations.

**Comment by peacekeeper** [comment]
I'm not so sure if this is a good idea. What about features like service endpoints or alsoKnownAs? I think those should remain in DID Core, since they don't really have much to do with Data Integrity. And then wouldn't we end up with a situation where some parts of DID documents (aka "controller documents") are defined in one spec, and other parts in another spec?

**Comment by msporny** [comment]
> I'm not so sure if this is a good idea. What about features like service endpoints or alsoKnownAs? I think those should remain in DID Core, since they don't really have much to do with Data Integrity.

Yes, agreed. My initial comment wasn't clear... the /base definition/ could come from Data Integrity, but we could easily layer stuff like service endpoints on top in the DID Core spec. There is an argument for `alsoKnownAs` going either in Data Integrity or in DID Core. Either would probably be fine, there.

> And then wouldn't we end up with a situation where some parts of DID documents (aka "controller documents") are defined in one spec, and other parts in another spec?

Yes, but that dynamic has always been the case -- a number of DID Method specifications extend controller documents and add features to their DID Documents that other DID Document don't have.

**Comment by OR13** [comment]
The main feature that "controller documents" have over "did documents" is they work with regular URLs, and `application/json` media types.... IMO this should be fixed with a small specification that can be cited from all the specs that need a generic definition of a controller document, including vc-data-model, did-core, data-integrity and vc-jose-cose.

**Comment by selfissued** [comment]
As we discussed on the WG call, this should be in a document in the VC working group that is independent of the securing specifications.  This is being tracked by https://github.com/w3c/vc-jose-cose/issues/140.

This is not an issue that the DID WG can address.  This issue should be closed on that basis.

**Comment by OR13** [comment]
VCWG is not likely to get to CR by filing issues in DID Core, we need address this in the active WG that is blocked by it.

**Comment by iherman** [comment]
Isn't this superseded by #854?

**Comment by msporny** [comment]
> Isn't this superseded by #854?

Yes, closing.
